frontstage/views/surveys/help/technical_help.py

```diff
@@ -40,6 +42,7 @@ def get_send_help_technical_message_page(session):
     option = request.args.get("option", None)
     subject = subject_text_mapping.get(option, None)
     breadcrumbs = breadcrumb_text_mapping.get(option, None)
+    logger.info("Request at start: " + str(request))
```

Comment:
Do we need this?

Reply:
Nope. Accidentally added for test purposes.